browser/bundles/o-errors.js

```diff
@@ -1,3 +1,12 @@
+// for safety polyfill window.console
+if (!window.console) {
+	window.console = {};
+	const methods = ['info', 'log', 'warn', 'error'];
+	for (let i = 0; i < methods.length; i++) {
+		window.console[methods[i]] = function () {};
```

Comment:
rather than this loop could it just not be 
```
if (!window.console) {
	window.console = {
		log: function () { },
		info: function () { },
		warn: function () { },
		error: function () { },
	};
}
```

Reply:
consider it copy pasted